mavlink/test_mission_ended.py

```diff
@@ -0,0 +1,137 @@
+"""
+Test for drone's destination at final waypoint by uploading mission and monitoring it.
+"""
+import sys
+import time
+
+import dronekit
+
+from mavlink.modules import flight_controller
+
+
+DELAY_TIME = 1.0  # seconds
+MISSION_PLANNER_ADDRESS = "tcp:127.0.0.1:14550"
+TIMEOUT = 1.0  # seconds
+
+MAVLINK_TAKEOFF_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_TAKEOFF_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_TAKEOFF
+MAVLINK_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT
+
+ALTITUDE = 10  # metres
+ACCEPT_RADIUS = 10  # metres
+
+
+# TODO: This function is to be removed when Dronekit-Python interfaces are
+# moved from pathing repository.
+def upload_mission(controller: flight_controller.FlightController,
+                   waypoints: "list[tuple[float, float, float]]") -> bool:
```

Comment:
I just realized that `FlightController` already has `upload_commands()` as a method. In the interest of time and that this code will be removed in the future, I will not block the PR on this particular matter.